### PR TITLE
Add jupyterhub route definitions

### PR DESCRIPTION
**Pull request by hemajv**
cc @HumairAK @anishasthana

### CODE REVIEW

**Review by anishasthana — APPROVED** [review]

File: jupyterhub/jupyterhub/overlays/prod/jupyterhub-route.enc.yaml [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+metadata:
+    annotations:
+        kubernetes.io/tls-acme: "true"
+    creationTimestamp: '2019-05-28T02:53:46Z'
+    labels:
+        app: jupyterhub
+    name: jupyterhub
+    namespace: dh-prod-jupyterhub
+    resourceVersion: "35432909"
+    selfLink: /apis/route.openshift.io/v1/namespaces/dh-prod-jupyterhub/routes/jupyterhub
```
Comment (anishasthana) [inline jupyterhub/jupyterhub/overlays/prod/jupyterhub-route.enc.yaml]: should this be there>

**Review by anishasthana — CHANGES_REQUESTED** [review]

**Review by anishasthana — CHANGES_REQUESTED** [review]
This is actually to the wrong branch

File: jupyterhub/jupyterhub/overlays/prod/jupyterhub-route.enc.yaml (hunk 1 above)
Comment (hemajv) [reply jupyterhub/jupyterhub/overlays/prod/jupyterhub-route.enc.yaml]: updated

File: jupyterhub/overlays/prod/kustomization.yaml [hunk 2]
```diff
@@ -3,4 +3,7 @@ resources:
 - ../../bases/jupyterhub
 - ../../bases/custom-images
 - ../../bases/custom-profiles
+- ../../bases
```
Comment (anishasthana) [inline jupyterhub/overlays/prod/kustomization.yaml]: is this correct?

File: jupyterhub/overlays/prod/kustomization.yaml [hunk 3]
```diff
@@ -3,4 +3,7 @@ resources:
...
 - membership.yaml
+generators:
+        - ./secret-generator.yaml
```
Comment (HumairAK) [inline jupyterhub/overlays/prod/kustomization.yaml]: remove indents (see the resources field format above it)

File: jupyterhub/overlays/prod/kustomization.yaml (hunk 2 above)
Comment (hemajv) [reply jupyterhub/overlays/prod/kustomization.yaml]: fixed

File: jupyterhub/overlays/prod/kustomization.yaml (hunk 3 above)
Comment (hemajv) [reply jupyterhub/overlays/prod/kustomization.yaml]: fixed

**Review by anishasthana — APPROVED** [review]

**Review by anishasthana — APPROVED** [review]

**Review by HumairAK — APPROVED** [review]

**Pull request merged**